wfsim/strax_interface.py

```diff
@@ -359,7 +342,7 @@ def __init__(self, config):
         log.debug(f'Raw data is set')
         self.record_buffer = np.zeros(5000000,
                                       dtype=strax.raw_record_dtype(samples_per_record=strax.DEFAULT_RECORD_LENGTH)) # 2*250 ms buffer
-        self.truth_buffer = np.zeros(10000, dtype=instruction_dtype + truth_extra_dtype + [('fill', bool)])
+        self.truth_buffer = np.zeros(100000, dtype=instruction_dtype + truth_extra_dtype + [('fill', bool)])
```

Comment:
I think I have seen too that we need this

Reply:
Hey @jorana, please notice that you're reviewing changes from #125. Keita commited his changes of top of the branch being reviewed in there.

Reply:
Thanks Diego, I wondered as such as some of the lines changed looked a little out of place given the scope of this PR, thanks for clarifying 😄